dashboard/app/views/maker/discountcode.html.haml

```diff
@@ -0,0 +1,6 @@
+- data = {}
+- data[:eligibility_status] = { statusPD: true, studentCount: false }
+
+%script{src: minifiable_asset_path('js/maker/discountcode.js'), data: {discountcode: data.to_json}}}
```

Comment:
Is the `to_json` and subsequent parsing redundant here?  The `data` hash is already getting implicitly converted to a browser-friendly format, and decoded again when we access the dataset from the client.  Is that not a deep operation?

Reply:
Without the to_json, it does not look like this gets put into a browser-friendly format. This is the same pattern we use a bunch of other places.

That said,, I did remember on the JS side of the things we have a helper `getScriptData` I should be using.